pycontrails/core/models.py

```diff
@@ -553,11 +570,21 @@ def set_source_met(
             # take the first var name output from ensure_vars
             met_key = self.met.ensure_vars(var)[0]
 
+            q_method = self.params["interpolation_q_method"]
```

Comment:
Would be unnecessary if you put this value in `interp_kwargs`

Reply:
Thanks!

fe10a1df11aed5cedc1afe767701ef1a869b7059